Test/scripts/phpcs_meqp2.sh

```diff
@@ -4,14 +4,34 @@ set -e
 set -u
 set -x
 
-sudo composer self-update -q
 composer show -i
 
-echo "{\"http-basic\":{\"repo.magento.com\":{\"username\":\"${MAGENTO_PUBLIC_KEY}\",\"password\":\"${MAGENTO_PRIVATE_KEY}\"}}}" > $HOME/.composer/auth.json
-
 cd ..
+
 composer create-project magento/magento-coding-standard --stability=dev magento-coding-standard
 
 cd magento-coding-standard
 
-vendor/bin/phpcs ../project --standard=Magento2 --colors --severity=10
+vendor/bin/phpcs ../project --standard=Magento2 --colors --severity=10 -p
+
+cd ../project
+export MAGENTO_VERSION="2.3.0"
```

Comment:
Why are we exporting this particular version?

Reply:
We already have script `Test/scripts/install_magento.sh`, which requires a specific magento version set before it runs. It doesn't matter which exact version we use.